### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by ashwinimone**
Readme for custom normalizations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+```
+
+## Usage
+Rest Normalization module helps to embed the entity referenced in the content, by altering the REST API. 
+
+For example, The Entity reference field in the REST Output provides the target_id, target_type and URI. In some cases, we need more data regarding the referenced entity other than the provided data. Rest Normalization helps to solve this problem by overriding the default values and embedding the entity data into the response.
+
+Create a .php file in the module directory with the following content. Replace the `<target_identifier>` with the project-specific entity reference field that needs to be embedded into the API response (for example: `paragraph-locaties-field_locations`)
+
+```sh
```
Comment (rolfvandekrol) [inline README.md]: This is not `sh` code, but `php`

File: README.md [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
-```
\ No newline at end of file
+```
+
+## Usage
+Rest Normalization module helps to embed the entity referenced in the content, by altering the REST API. 
+
+For example, The Entity reference field in the REST Output provides the target_id, target_type and URI. In some cases, we need more data regarding the referenced entity other than the provided data. Rest Normalization helps to solve this problem by overriding the default values and embedding the entity data into the response.
+
+Create a .php file in the module directory with the following content. Replace the `<target_identifier>` with the project-specific entity reference field that needs to be embedded into the API response (for example: `paragraph-locaties-field_locations`)
```
Comment (rolfvandekrol) [inline README.md]: Please translate `locaties` to `locations`, and add explanation which target identifiers are accepted and how they should be constructed.

File: README.md (hunk 1 above)
Comment (rolfvandekrol) [reply README.md]: Could you make clear where these code should be placed and how the name of the class relates to the name of the module it is placed in.

**Review by rolfvandekrol — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ashwinimone) [reply README.md]: Could you explain the "how they should be constructed"?

Comment (rolfvandekrol) [reply README.md]: There are three types of valid target identifiers:

* the entity id of the referenced entity type
* `A-B` where `A` is the entity id of the entity type the field is on, and `B` is the name of the field
* `A-B-C` where `A` is the entity id of the entity type the field ion, `B` is the bundle where the field is on, and `C` is the name of the field.

This should be reworded and made a little more clear, but that's the gist. The logic where the valid target identifiers are constructed is here: https://github.com/hoppinger/rest_normalizations/blob/master/src/ComponentGenerator/EntityReferenceTargetFieldGenerator.php#L36-L42

**Pull request merged**